doc/book/replication/repl_leader_elect.rst

```diff
@@ -0,0 +1,201 @@
+  in the configuration reference.
+* ``election_timeout`` -- specifies the timeout between election rounds if the
+  previous round ended up with a split-vote. For the details, refer to the
+  :ref:`option description <cfg_replication-election_timeout>` in the configuration
+  reference.
+* ``replication_timeout`` -- reuse of the :ref:`replication_timeout <cfg_replication-replication_timeout>`
+  configuration option for the purpose of the leader election process.
+  Heartbeats sent by an active leader have a timeout after which a new election
+  is started. Heartbeats are sent once per <replication_timeout> seconds.
+  Default value is ``1``. The leader is considered dead if it hasn't sent any
+  heartbeats for the period of ``<replication_timeout> * 4``.
+* ``replication_synchro_quorum`` -- -- reuse of the :ref:`replication_synchro_quorum <cfg_replication-replication_synchro_quorum>`
+  option for purpose of configuring the election quorum. Default value is ``1``
+  meaning that each node becomes a leader immediately after voting for itself.
+  It is the best to set up this option value to the ``(<cluster size> / 2) + 1``.
+  Otherwise there is no a guarantee that there is only one leader at a time.
+
+Besides, it is important to take into account that
+being a leader is not the only requirement for a node to be writable.
+A leader node should have its :ref:`read_only <cfg_basic-read_only>` option set
+to ``false`` (``box.cfg{read_only = false}``),
+and its :ref:`connectivity quorum <cfg_replication-replication_connect_quorum>`
+should be satisfied (``box.cfg{replication_connect_quorum = <count>}``)
+or disabled (``box.cfg{replication_connect_quorum = 0}``).
+Nothing prevents from setting the ``read_only`` option to ``true``,
+but the leader just won't be writable then. The option doesn't affect the
+election process itself, so a read-only instanc still can vote and become
+a leader.
+
+.. _repl_leader_elect_monitoring:
+
+--------------------------------------------
+Monitoring
+--------------------------------------------
+
+To monitor the current state of a node regarding the leader election, you can
+use the ``box.info.election`` function. For the detailed description,
+refer to the :ref:`function description <box_info_election>`.
+
+**Example:**
+
+.. code-block:: console
+
+   tarantool> box.info.election
+   ---
+   - state: follower
+     vote: 0
+     leader: 0
+     term: 1
+   ...
+
+As for logging, the election implementation based on Raft logs all its actions
+with the ``RAFT:`` prefix. The actions are new Raft message handling,
+node state changing, voting, term bumping, and so on.
+
+.. _repl_leader_elect_important:
+
+--------------------------------------------
+Important notes
+--------------------------------------------
+
+Leader election won't work properly if the election quorum is set less or equal
+than ``<cluster size> / 2`` because in that case the split-vote can happen (two
```

Comment:
since before we use the 'split-vote' for describing of a case where no majority is collected, I think here it should be used as 'split-vote can lead to a state when two leaders elected at once'

Reply:
corrected